packages/services/src/view-service/auctions.ts

```diff
@@ -65,12 +65,18 @@ export const auctions: Impl['auctions'] = async function* (req, ctx) {
     let state: DutchAuctionState | undefined;
     if (queryLatestState) state = (await querier.auction.auctionStateById(id))?.state;
 
+    const outstandingReserves = await indexedDb.getAuctionOutstandingReserves(id);
```

Comment:
Should this be moved into the if clause given it's only needed if `!!value.auction || state` is true?

Reply:
Good catch. Fixed.